CMakeLists.txt

```diff
@@ -33,12 +33,17 @@ set(VISIBLE_cmake_modules_IND_GUI_ENTRIES OPENCMISS_CMAKE_MODULES_ROOT)
 
 set(SETUP_PROJECTS CMAKE_MODULES MANAGE)
 
-set(MANAGE_ORG OpenCMISS)
-set(CMAKE_MODULES_ORG OpenCMISS)
+set(OPENCMISS_SETUP_ORGANISATION OpenCMISS CACHE STRING "The GitHub organisation to use for setup i.e., for the cmake_modules and manage repositories.")
+set(OPENCMISS_SETUP_BRANCH develop CACHE STRING "The cmake_modules and manage repositories branch to use for setup.")
```

Comment:
These two options should be placed into the options file and documented fully.

Reply:
The setup organisation and branch are used to set the manage organisation and branch etc. towards the top of CMakeLists.txt and before the Options.cmake file is included. This would require the Options.cmake file to be included towards the start of CMakeLists.txt?

Reply:
Or alternatively move this section down below the Options inclusion.  This section was placed at the top to enable easy editing, but if there is a GUI editable version this becomes less important.  Anything that may be edited that isn't a GUI dependent variable should be towards the top of the file.